OJP/OJP_FareSupport.xsd

```diff
@@ -408,9 +432,9 @@
 					</xs:annotation>
 				</xs:element>
 			</xs:choice>
-			<xs:element name="EntitlementProduct" type="EntitlementProductRefStructure" minOccurs="0" maxOccurs="unbounded">
+			<xs:element name="EntitlementProducts" type="EntitlementProductListStructure" minOccurs="0" maxOccurs="unbounded">
```

Comment:
The structure allowed multiple `EntitlementProduct` elements to exist beneath each other before the change due to the `maxOccurs="unbounded"`. So you are now introduction an explicit list element to group them.
Was the motivation only to have multiple EntitlementProducts and you overlooked the `maxOccurs="unbounded"` or was it intentional to have them in a list? If for the first option this change would not be necessary, if for the second option the `maxOccurs="unbounded"` on the list must be removed because otherwise there could be multiple groups.